dotnet/src/SemanticKernel.Skills/Skills.OpenAPI/Model/RestApiOperationParameter.cs

```diff
@@ -37,6 +37,16 @@ internal class RestApiOperationParameter
     /// </summary>
     public RestApiOperationParameterLocation Location { get; }
 
+    /// <summary>
+    /// The parameter style - defines how multiple values are delimited.
+    /// </summary>
+    public RestApiOperationParameterStyle? Style { get; }
+
+    /// <summary>
+    /// Type of array item for parameters of "array" type.
+    /// </summary>
+    public string? ItemsType { get; }
```

Comment:
I would name it more explicitly, like `ArrayItemsType` or `ArrayType`. Name `ItemsType` doesn't clearly describe that this property is related to array items only.

Reply:
Yep, make sence.